offsets.py

```diff
@@ -150,7 +151,7 @@ class OffsetsFirmware_903:
     POP_RCX_RET = 0xFFFFFFFF822445E7 # 59 C3
 
     # pop r8 ; pop rbp ; ret
-    POP_R8_POP_RBP_RET = 0xFFFFFFFF822AB4DD # 47 58 5D C3
+    POP_R8_POP_RBP_RET = 0xffffffff8237dd7d # C5 FB 11 47 58 5D C3
```

Comment:
In fact you don't want the "C5 FB 11" part, so the comment is kind of misleading..

Reply:
@zecoxao

Reply:
@loskutov I guess I should go to bed at this point. Man, how did I do that? 🤦‍♂️